unit_tests/log_transfer/test_main_log_lsa_utils.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#define CATCH_CONFIG_MAIN
+
+#include "catch2/catch.hpp"
+
+#include "log_lsa.hpp"
+#include "log_lsa_utils.hpp"
+#include "packer.hpp"
+
+class test_env
+{
+  public:
+    test_env ();
+    ~test_env ();
+
+    void run_test ();
+};
+
+test_env::test_env ()
+{
+}
+
+test_env::~test_env ()
+{
+}
+
+void test_env::run_test ()
+{
+  log_lsa lsa;
+  lsa.pageid = 4;
+  lsa.offset = 8;
+  char buffer[8];
+
+  cubpacking::packer packer (buffer, 8);
+  cublog::lsa_utils::pack (lsa, packer);
+
+  cubpacking::unpacker unpacker (buffer, 8);
+  log_lsa unpacked_lsa;
+  cublog::lsa_utils::unpack (unpacker, unpacked_lsa);
+
+  REQUIRE (lsa.pageid == unpacked_lsa.pageid);
+  REQUIRE (lsa.offset == unpacked_lsa.offset);
+
+  int64_t big_int = lsa;
+  log_lsa new_lsa (big_int);
+
+  REQUIRE (lsa == new_lsa);
+}
+
+void do_test (test_env &env)
+{
+  env.run_test ();
+}
+
+TEST_CASE ("First test", "")
+{
+  test_env env;
+  do_test (env);
+}
```

Comment:
Having a test_env for this simple test seems overkill. 
```suggestion
TEST_CASE ("First test", "")
{
  log_lsa lsa;
  lsa.pageid = 4;
  lsa.offset = 8;
  char buffer[8];

  cubpacking::packer packer (buffer, 8);
  cublog::lsa_utils::pack (lsa, packer);

  cubpacking::unpacker unpacker (buffer, 8);
  log_lsa unpacked_lsa;
  cublog::lsa_utils::unpack (unpacker, unpacked_lsa);

  REQUIRE (lsa.pageid == unpacked_lsa.pageid);
  REQUIRE (lsa.offset == unpacked_lsa.offset);

  int64_t big_int = lsa;
  log_lsa new_lsa (big_int);

  REQUIRE (lsa == new_lsa);
}
```